### PR TITLE
Add BDOGE Coin Token

### PR DESCRIPTION
**Pull request by yopkub**

### CODE REVIEW

File: tokens/f555c46bad0731d080c9381d7fff6f82839946a66bd070d185e1ea2f.BlackDogeCoin.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "policyId": "f555c46bad0731d080c9381d7fff6f82839946a66bd070d185e1ea2f",
+    "name": "BlackDogeCoin",
```
Comment (MuesliSwapTeam) [inline tokens/f555c46bad0731d080c9381d7fff6f82839946a66bd070d185e1ea2f.BlackDogeCoin.json:3]:
```suggestion
    "name": "BDOGE",
```

This needs to be your asset name. If I see correctly, that is also "BDOGE" (https://cardanoscan.io/token/f555c46bad0731d080c9381d7fff6f82839946a66bd070d185e1ea2f42444f4745)

**Comment by MuesliSwapTeam** [comment]
Thank you for the submission! Please resolve the open issues and add the image in this PR, then we can go ahead and list it :)

**Comment by yopkub** [comment]
@MuesliSwapTeam Already change name to BDOGE, please check

**Pull request merged**